src/psyclone/psyir/symbols/symboltable.py

```diff
@@ -83,6 +83,39 @@ def __init__(self, node=None):
                 "Schedule or a Container but found '{0}'."
                 "".format(type(node).__name__))
         self._node = node
+        # The default visibility of symbols in this symbol table. (Required
+        # when we have symbols of UnknownType since their associated
+        # declaration may or may not contain visibility information.)
+        self._default_visibility = None
+        if default_visibility is not None:
+            self.default_visibility = default_visibility
+
+    @property
+    def default_visibility(self):
+        '''
+        :returns: the default visibility of symbols in this table.
+        :rtype: :py:class:`psyclone.psyir.symbols.Symbol.Visibility`
+        '''
+        return self._default_visibility
+
+    @default_visibility.setter
+    def default_visibility(self, vis):
+        '''
+        Sets the default visibility of symbols in this table. (Only used for
+        Symbols of UnknownType since their visibility may or may not be
```

Comment:
I would leave "Sets the default visibility parameter of this table." and don't mention how this is used externally to the class.

Reply:
Yes, done.